src/benchmarklib/tpch/tpch_queries.cpp

```diff
@@ -615,7 +615,9 @@ const char* const tpch_query_15 =
 
       SELECT s_suppkey, s_name, s_address, s_phone, total_revenue FROM supplier, revenue
       WHERE s_suppkey = supplier_no AND total_revenue = (SELECT max(total_revenue)
-      FROM revenue) ORDER BY s_suppkey;)";
+      FROM revenue) ORDER BY s_suppkey;
+
+      drop view revenue;)";
```

Comment:
Minor: `DROP VIEW revenue` to keep case the same across the file?

Reply:
No idea why github marks this (and the next two) as outdated. I think they are not.